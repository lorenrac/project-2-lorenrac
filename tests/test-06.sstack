// Try a bunch of scopes.
PRINT "Scope 0"
{
  PRINT "Scope 1"
  {
    PRINT "Scope 2"
  }
  PRINT "Scope 1 again"
}
PRINT "Scope 0 again"
