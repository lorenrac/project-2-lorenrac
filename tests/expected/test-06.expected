Scope 0
Scope 1
Scope 2
Scope 1 again
Scope 0 again
